src/bebb/main.mo

```diff
@@ -163,8 +163,8 @@ actor {
   private func createEntity(caller : Principal, entityToCreate : Entity.EntityInitiationObject) : async Text {
 
     Debug.print("In create Entity");
-    let canisterId = IndexCanister.getCanisterIdsIfExists("entity#")
-    Debug.print("Response: " # canisterId);
+    //let canisterId = IndexCanister.getCanisterIdsIfExists("entity#") //getCanisterIdsIfExists is a private func
```

Comment:
this wouldn't compile as it's not a public function on IndexCanister

Reply:
oh good to know